lib/webpacker/manifest.rb

```diff
@@ -54,22 +54,32 @@ def missing_file_from_manifest_error(bundle_name)
 4. Your webpack configuration is not creating a manifest.
 Your manifest contains:
 #{JSON.pretty_generate(@data)}
-      MSG
-    end
+    MSG
+  end
 
-    def data
-      if config.cache_manifest?
-        @data ||= load
-      else
-        refresh
-      end
+  def data
+    if config.cache_manifest?
+      @data ||= load
+    else
+      refresh
     end
+  end
 
-    def load
-      if config.public_manifest_path.exist?
-        JSON.parse config.public_manifest_path.read
-      else
-        {}
+  def load
+    if config.public_manifest_path.exist?
+      data = JSON.parse config.public_manifest_path.read
+      #if the build was done on a windows box manifest.json will have backslashes as keys
+      #we will ensure lookups continue to succeed
+      data.keys.each do |key|
```

Comment:
Shall we wrap this code inside? `if Gem.win_platform?`

Reply:
No, it has nothing to do where the code is running.  Only where the build was run.

Reply:
Oh yes ofcourse 👍